CMakeLists.txt

```diff
@@ -418,12 +418,13 @@ if( BUILD_RUBY )
     # tar -zcvf ruby_2_5_1_msvc2017_static.tar.gz Ruby-install    
     ExternalProject_Add(Ruby
       DEPENDS ZLib FFI
-      URL https://codeload.github.com/ruby/ruby/tar.gz/v2_5_1
-      URL_MD5 98b550a05fec0f7a43a22d6c37404fd4
+      URL https://codeload.github.com/ruby/ruby/tar.gz/v2_5_5
+      URL_HASH SHA256=A3BA7160C7E0CE03865CDABA9B7C26974A8A0FC23FC953269E0463C5ACAFB7A1
       BUILD_IN_SOURCE 1
       # DLM: ruby_2_5_1_msvc2017_static_v3.tar.gz on s3 was built with experimental Ruby.win.wopen.patch patch from ruby_patches branch, not using it for now
       #PATCH_COMMAND ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.patch && ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.win.patch && ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.win.wopen.patch
-      PATCH_COMMAND ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.patch && ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.win.patch && ${PATCH_EXE} 
+      PATCH_COMMAND ${PATCH_EXE} -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.patch && ${PATCH_EXE}  -p1 < ${CMAKE_SOURCE_DIR}/dependencies/ruby/Ruby.win.patch && ${PATCH_EXE} 
```

Comment:
Did something important change on this line?

Reply:
No, and that shouldn't have been a commit, I tried to make sure that minor (accidental) diff didn't show up in the diffs.

Reply:
Removed empty space.